user-guide/Advanced_Functionality/Databases/Indexing_database/OpenSearch_Database/Installing_OpenSearch_database.md

```diff
@@ -6,115 +6,237 @@ uid: Installing_OpenSearch_database
-  #
-  # ----------------------------------- Paths ------------------------------------
-  #
-  # Path to directory where to store the data (separate multiple locations by comma):
-  #
-  path.data: /var/lib/opensearch
-  #
-  # Path to log files:
-  #
-  path.logs: /var/log/opensearch
-  # ---------------------------------- Network -----------------------------------
-  #
-  # Set the bind address to a specific IP (IPv4 or IPv6), best is to use the real-ip of the node:
-  #
-  network.host: 166.206.186.146
-  
-  network.publish_host: 166.206.186.146
-  #
-  # Set a custom port for HTTP:
-  #
-  http.port: 9200
-  #
-  # For more information, consult the network module documentation.
-  #
-  # --------------------------------- Discovery ----------------------------------
-  #
-  # Pass an initial list of hosts to perform discovery when this node is started:
-  # The default list of hosts is ["127.0.0.1", "[::1]"]
-  #
-  discovery.seed_hosts: ["166.206.186.146","166.206.186.147","166.206.186.148"]
-  
-  discovery.type: zen
+    `$ sudo ufw allow from [IP node 2] to [IP node 1] proto tcp port 9200,9300`
+
+    `$ sudo ufw allow from [IP node 3] to [IP node 1] proto tcp port 9200,9300`
+
+  - Commands node 2:
+
+    `$ sudo ufw allow from [IP node 1] to [IP node 2] proto tcp port 9200,9300`
+
+    `$ sudo ufw allow from [IP node 3] to [IP node 2] proto tcp port 9200,9300`
   
-  node.max_local_storage_nodes: 3
-  indices.query.bool.max_clause_count: 2147483647
+  - Commands node 3:
 
-  ```
+    `$ sudo ufw allow from [IP node 1] to [IP node 3] proto tcp port 9200,9300`
 
-- If you want a node to be a **data node**, add the following configuration in *OpenSearch.yml*:
+    `$ sudo ufw allow from [IP node 2] to [IP node 3] proto tcp port 9200,9300`
 
-  ```yml
-  node.roles: [ data, ingest ]
-  ```
+- Make sure all DMAs in the DMS can connect to port 9200 and 9300:
 
-- If you want a node to be the **cluster manager node** (a.k.a. the master node), add the following configuration in *OpenSearch.yml*:
+  - Connectivity for DMA 1 (with OpenSearch Dashboards connection allowed):
 
-  ```yml
-  node.roles: [ cluster_manager ]
-  ```
+    `$ sudo ufw allow from [IP node DMA 1] to [IP node 1] proto tcp port 9200,9300,5601`  
+
+    `$ sudo ufw allow from [IP node DMA 1] to [IP node 2] proto tcp port 9200,9300,5601`  
+
+    `$ sudo ufw allow from [IP node DMA 1] to [IP node 3] proto tcp port 9200,9300,5601`  
+
+  - Connectivity for DMA 2:
+
+    `$ sudo ufw allow from [IP node DMA 2] to [IP node 1] proto tcp port 9200,9300`  
+
+    `$ sudo ufw allow from [IP node DMA 2] to [IP node 2] proto tcp port 9200,9300`  
+
+    `$ sudo ufw allow from [IP node DMA 2] to [IP node 3] proto tcp port 9200,9300`  
+
+  - And so on.
+
+### Mounting the data drive
+
+In most cases, a separate hard drive will be used to store database data. The data drive has to be mounted and path to the data directory has to be specified in the Openseconfiguration file (see below).
```

Comment:
Is "Openseconfiguration" a typo or is this actually the name of a file?

Also, could you replace the "see below" reference with an internal cross-reference? That would make it easier for the user to know which of the sections below you actually mean.

Reply:
This was a typo. Corrected and added a cross-reference as suggested.